wooey/settings.py

```diff
@@ -15,4 +15,6 @@ def get(key, default):
 WOOEY_REGISTER_URL = get('WOOEY_REGISTER_URL', '/accounts/register/')
 WOOEY_SHOW_LOCKED_SCRIPTS = get('WOOEY_SHOW_LOCKED_SCRIPTS', True)
 WOOEY_EPHEMERAL_FILES = get('WOOEY_EPHEMERAL_FILES', False)
-WOOEY_DEFAULT_SCRIPT_GROUP = get('WOOEY_DEFAULT_SCRIPT_GROUP', _('Wooey Scripts'))
\ No newline at end of file
+WOOEY_DEFAULT_SCRIPT_GROUP = get('WOOEY_DEFAULT_SCRIPT_GROUP', _('Wooey Scripts'))
```

Comment:
I wonder if we should change the default script group name to "Scripts" to remove the use of the name in the admin.